src/shared/types/Product.ts

```diff
@@ -0,0 +1,12 @@
+export interface Product {
```

Comment:
Rename it to ProductCard, because we already have Product - it is name of page: /src/pages/Product/index.tsx